addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -594,6 +596,17 @@ var SnippetEditor = Widget.extend({
         }
         return show;
     },
+    /**
+     * @private
+     * @param {Object} $el
+     */
+    _getActivatedTarget: function ($el) {
```

Comment:
- The name is confusing, it should be something like `_getPostRemovalElementToActivate`, but better: do not create a function for that. It is only called once and it would be easier to understand if the code was directly where it is used with a comment.